dictionaries/top-10000-project-gutenberg-words.json

```diff
@@ -4377,7 +4377,7 @@
 "PW*P/KR*P": "BC",
 "AEUB/HAPL": "Abraham",
 "THAUFL": "thoughtful",
-"TRAFL/HRER/A*U": "traveller",
+"TRA*FL/HRER": "traveller",
```

Comment:
As above

Reply:
Yep. As per [this comment](https://github.com/didoesdigital/steno-dictionaries/pull/276#discussion_r406086750), `TRAFL/HRER` is already in the dictionaries as "traveler".